Makefile

```
@@ -5,6 +5,10 @@ build:
 test:
 	go test -v -race ./... -timeout 40m
 
+# traces-dataset.sz is used by ./pkg/tests/end_to_end_tests/ingest_trace_test.go
+download-trace-dataset:
+	curl -L -o ./pkg/tests/testdata/traces-dataset.sz https://github.com/timescale/promscale-test-data/raw/main/traces-dataset.sz
+
 e2e-test:
```

```suggestion
e2e-test: pkg/tests/testdata/traces-dataset.sz
```

This would allow using the previously defined target and reduce test execution to just `make e2e-test`.